### PR TITLE
Add windows .exe extension to the path

### PR DESCRIPTION
**Pull request by Laykou**
When running on the windows machine the binary file is called `phantomjs.exe` by default.
The **.exe** extension should be added to the path.

Refs #16

### CODE REVIEW

**Comment by grantlucas** [comment]
Thanks for the PR @Laykou! This addition is definitely needed due to the way PhantomJS installer works. Added a comment to the code changes for one thing.

File: src/Phantoman.php [hunk 1]
```diff
@@ -34,6 +34,11 @@ public function __construct($config, $options)
             $this->config['path'] = "vendor/bin/phantomjs";
         }
 
+        // Add .exe extension if running on the windows
+        if ($this->isWindows() && file_exists(realpath($this->config['path'] . '.exe'))) {
+            $this->config['path'] .= '.exe';
+        }
```
Comment (grantlucas) [inline src/Phantoman.php]: This `.exe` appending needs to happen after the "is directory" check on line 43 in order to ensure that windows setups that use *just* the folder will get the correct .exe name.

### Steps that would happen on windows to support the old directory config

1. Check if the configured path is a directory
    - If so, append `/phantomjs` to the end of the path
2. Check if we're on windows
    - If so, append `.exe` to the end of the path.

From `vendor/bin`, we should end up with `vendor/bin/phantomjs.exe` on Windows.

Can you make that change and let me know if setting the path to `vendor/bin` on Windows works for you? I'd like to ensure the old methods work. You will get the deprecation warning but that's what we want in that case.

After you ensure that works, you can set it to the normal `vendor/bin/phantomjs` etc.

Thank you for you help!

Comment (Laykou) [inline src/Phantoman.php]: Obviously the order was wrong. Now it works as you described. I've tested with `vendor/bin`, `vendor/bin/phantomjs`, `vendor/bin/phantomjs.exe` and empty (using the default value). All paths **works** as expected.

So I think you can merge this.

Comment (grantlucas) [inline src/Phantoman.php]: :tada: Awesome. Thanks for testing those cases!

**Pull request merged**